index.html

```diff
@@ -101,127 +92,136 @@ <h2 class="header-text">My Recent Works</h2>
-                <h2 class="card-header">Professional Art <br> Printing Data</h2>
-                <p class="card-description">
-                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-                </p>
-                <div>
-                    <ul class="card-tags">
-                        <li><button class="tag blur">html</button></li>
-                        <li><button class="tag blur">bootstrap</button></li>
-                        <li><button class="tag blur">Ruby</button></li>
-                    </ul>
+                <!-- Fifth -->
+                <div class="work-card">
+                    <h2 class="card-header">Professional Art <br> Printing Data</h2>
+                    <p class="card-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <div>
+                        <ul class="card-tags">
+                            <li><button class="tag blur">html</button></li>
+                            <li><button class="tag blur">bootstrap</button></li>
+                            <li><button class="tag blur">Ruby</button></li>
+                        </ul>
+                    </div>
+                    <button class="btn-max-width hide-btn">See Project</button>
+                </div>
+                <!-- sixth -->
+                <div class="work-card">
+                    <h2 class="card-header">Professional Art <br> Printing Data</h2>
+                    <p class="card-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <div>
+                        <ul class="card-tags">
+                            <li><button class="tag blur">html</button></li>
+                            <li><button class="tag blur">bootstrap</button></li>
+                            <li><button class="tag blur">Ruby</button></li>
+                        </ul>
+                    </div>
+                    <button class="btn-max-width hide-btn">See Project</button>
                 </div>
-                <button class="btn-max-width">See Project</button>
             </div>
         </div>
     </div>
     </section>
     <section class="about-section">
-        <div>
-            <h1  class="primary-text">about me</h1>
-        </div>
-        <div>
-            <h2 class="description">
-            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </h2>
+        <div class="about-container">
+            <div>
+                <h1  class="primary-text">about me</h1>
```

Comment:
- Please can you correct this to `About me`? 👍🏽

Reply:
Changed the letter for the header.